kind.sh

```diff
@@ -0,0 +1,247 @@
+#!/bin/sh
+
+set -o errexit
+###############
+# User config #
+###############
+
+# How to use this?
+# Define env vars KIND_IMAGE, KIND_CLUSTER_NAME (or let them be the default),
+# and the KURED_IMAGE_DEST (the public repo which will contain the dev version
+# of the image to test).
+# The process will build kured (using make image), tag the image with
+# KURED_IMAGE_DEST, publish it somewhere, spin up a kind cluster, install
+# kured in it, create a reboot sentinel file, track if reboot is happening
+# by watching the kubernetes api.
+# The resources files/kind cluster are destroyed at the end of the process
+# unless DO_NOT_TEARDOWN is not empty
+# For CI, you should probably set -x before running this script.
+
+#Define the image used in kind, for example:
+#KIND_IMAGE=kindest/node:v1.18.2
+#KIND_IMAGE=kindest/node:v1.18.0@sha256:0e20578828edd939d25eb98496a685c76c98d54084932f76069f886ec315d694
+#KIND_IMAGE=kindest/node:v1.17.0@sha256:9512edae126da271b66b990b6fff768fbb7cd786c7d39e86bdf55906352fdf62
+#KIND_IMAGE=kindest/node:v1.16.4@sha256:b91a2c2317a000f3a783489dfb755064177dbc3a0b2f4147d50f04825d016f55
+KIND_IMAGE=${KIND_IMAGE:-"kindest/node:v1.18.2"}
+
+# desired cluster name; default is "kured"
+KIND_CLUSTER_NAME="${KIND_CLUSTER_NAME:-kured}"
+
+KURED_IMAGE_DEST=${KURED_IMAGE_DEST:-"docker.io/evrardjp/kured:latest"}
```

Comment:
Maybe complain if `KURED_IMAGE_DEST` is not set instead?

Reply:
Ok.